common/src/main/java/com/sequenceiq/cloudbreak/eventbus/Event.java

```diff
@@ -0,0 +1,81 @@
+package com.sequenceiq.cloudbreak.eventbus;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+public class Event<T> {
```

Comment:
could you add toString impl? as I see the original class have it

Reply:
Sure.